assets/disclosure.css

```diff
@@ -13,7 +13,7 @@
 }
 
 .disclosure__list {
-  border: 1px solid rgba(var(--color-foreground), 0.2);
+  border: var(--popup-drawer-border-width) solid rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
```

Comment:
the border is a bit wonky with the scrollbar. ![](https://screenshot.click/21-11-b4e4z-y2ytz.png)

Reply:
@ludoboludo Do you mind sharing your exact settings? Cant seem to reproduce it https://screenshot.click/12-31-319qa-ljv3k.mp4

Reply:
Thanks, Ludo

It looks like the bar will just overflow because there isnt enough space 🤔 Not sure if we should add more padding to it or just leave as is. https://screenshot.click/15-08-jotgt-70jhm.mp4

cc @melissaperreault

Reply:
Hmm yeah it seems like one option would be to have a div wrapper around it with `overflow: hidden` but it might mess up the current styling applied and CSS specificity 🤔 
There is also the option of trying to style the scroll bar and add a border-radius to it but it won't work on all browsers.